javascript/javascript1/week1/goodboy.js

```diff
@@ -0,0 +1,13 @@
+let dogYearOfBirth = 2017;
+
+let dogYearFuture = 2027;
+
+let dogYear = dogYearFuture - dogYearOfBirth;
+
+if (shouldShowResultInDogYears = true) {
```

Comment:
Hi Saru, i think we need to talk ☎️  let's catch eachother on slack

Reply:
Done with the changes